src/app/components/data-entry-grid/index.html

```diff
@@ -107,6 +107,10 @@ <h2>ag-Grid styles</h2>
         To add the SKY UX styles for ag-Grid to your SPA, include <sky-code>"@skyux/ag-grid/css/sky-ag-grid.css"</sky-code> in the <sky-code>styles</sky-code> section of your <sky-code>skyuxconfig.json</sky-code>. For editable grids, include the class <sky-code>sky-ag-grid-editable</sky-code> on the <sky-code>ag-grid-angular</sky-code> element.
       </p>
 
+      <p>
+        To add ag-Grid modules alongside SKY UX modules, include <sky-code>SkyAgGridModule</sky-code> as an export in the <sky-code>app-sky.module.ts</sky-code> file. Next, in the <sky-code>app-extras.module.ts</sky-code> file, add <sky-code>AgGridModule.withComponents([])</sky-code> to the imports section and <sky-code>AgGridModule</sky-code> to the exports section.
```

Comment:
I added Erika as a reviewer so that she can weigh in on these changes.

I don't think this info belongs in the "ag-Grid styles" section. I think it belongs in the "Modules & installation" section. 

Also, I think the instructions here need to take into account multiple scenarios. I mean, I think the first sentence that you are adding is specifically for SPAs that are using the SKY UX SDK, and I'm pretty sure other SPAs, such as libraries and third-party SPAs, need to do something similar but not in `app-sky.module.ts`. So I think we need to modify that first sentence to reflect that. I think it needs to be something more like: 
"After installation, you need to make any exports from `SkyAgGridModule` available to your SPA by adding the module to the exports section of your SPA's main module. If you are using the SKY UX SDK, add `SkyAgGridModule` as an export in your `app-sky.module.ts` file."
(This feedback is based on working with Steve on the installation section in the past. We definitely need to get Erika and/or another dev on the team to confirm this feedback.)

I'm not sure if the second sentence needs similar updates to account for scenarios where SPAs aren't using the SKY UX SDK, but I would assume so. We need to check with a dev on that. ... Also, I think we should make `exports` and `imports` code snippets in the sentence because that's how we format elsewhere when referring to sections in the code. ... And maybe we should link to the agGrid documentation that Thomas referenced in his feedback (second bullet in the docs issue; we'd need to do some digging to find what docs he was referring to). ... And it sounded like Thomas was confused by the need to re-export `AgGridModule` so I wonder if Erika can help us provide a succinct explanation of why that step is necessary.